### PR TITLE
Fix #32: IE11/Edge Style and Layout Issues

### LINKED ISSUE #32: IE11/Edge Style and Layout Issues
Here's what it looks like on a Windows 8.1 VM in Internet Explorer 11:
win8 1-ie11

It looks similar in Edge on Windows 8.1.

### PR DESCRIPTION
**Pull request by decoy31**
To fix the layout issues shown in Issue #32, I decided to move away from using a flexbox style layout due to the [issues IE11 has with flexbox](http://caniuse.com/#search=flexbox). Instead, I went with a percentage width, inline-block style layout. Although not perfect, the inline-block style layout allowed for the highest amount of compatibility with the browsers this widget is described to support.

The most glaring style issue was the header text background color with a linear gradient. ~I had to remove the linear gradient from the related `h1`'s styles because [IE11 and Edge both do not support `background-clip: text`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-clip#Browser_compatibility).~ Because [IE11 and Edge both do not support `background-clip: text`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-clip#Browser_compatibility), I wrapped the related styles in a `@support` at-rule rule set which will disable the linear gradient on the header text for browsers that do no support `background-clip` (including IE11/Edge).

While I was testing my changes, I noticed there was a console error in IE11/Edge on the [modal demo page](https://widget.battleforthenet.com/demos/modal.html). Upon further investigation, I found there is a call to instantiate a new `URLSearchParams` object inside modal.html. Unfortunately IE11 and Edge do not have the `URLSearchParams` class defined, causing a console error to appear upon the demo page's load and breaking the use of the `theme` and `org` query string to change their respective widget properties in IE11/Edge. I created a rudimentary workaround which should suffice for the demo page's purposes.

**Microsoft Internet Explorer 11 on Windows 10 - Large Screen:**
ie11win10

**Microsoft Internet Explorer 11 on Windows 10 - Small Screen:**
ie11win10 - small screen

### CODE REVIEW

File: iframe/style.css [hunk 1]
```diff
@@ -327,9 +343,6 @@ body.stop h1 {
 
 body.slow h1,
 body.without h1 {
-  background: linear-gradient(#39ebfd, #6b74fb);
```
Comment (EdenSG) [inline iframe/style.css]: Perhaps a [feature query](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) could be used to enable the gradient if it's supported?

File: iframe/style.css [hunk 2]
```diff
@@ -1,14 +1,24 @@
+html {
+  font-size: 16px;
```
Comment (decoy31) [inline iframe/style.css]: This ensures `rem` values are always relative to a 16 pixel font size. While this is the default user agent font size for the `html` element in most browsers, in my experience, it's good to be explicit.

File: iframe/style.css (hunk 1 above)
Comment (decoy31) [inline iframe/style.css]: Nice! Learned something new today. Thanks!

I went ahead and wrapped the related style rules in a `@supports` at-rule set.

Comment (EdenSG) [inline iframe/style.css]: Awesome! Looks good.

**Comment by mikemorris** [comment]
Pulled most of these changes into https://github.com/fightforthefuture/battleforthenet-widget/pull/48, thanks!

**Pull request closed without merging**